### PR TITLE
update to 1.0.3

### PR DESCRIPTION
**Pull request by daltonbohning**
Update to version 1.0.3.
Update packaging to latest.

See https://github.com/daos-stack/mpifileutils-pkg/pull/9 for more details on the train of dependencies.

### CODE REVIEW

File: lwgrp.spec [hunk 1]
```diff
@@ -36,8 +36,8 @@
 %endif
 
 Name:		lwgrp
-Version:	1.0.2
-Release:	3.2%{?dist}
+Version:	1.0.3
+Release:	4%{?dist}
```
Comment (brianjmurrell) [inline lwgrp.spec]:
```suggestion
Release:	1%{?dist}
```

File: lwgrp.spec [hunk 2]
```diff
@@ -177,6 +177,9 @@ done
 %endif
 
 %changelog
+* Tue Jan 19 2021 Dalton A. Bohning <daltonx.bohning@intel.com> - 1.0.3-4
```
Comment (brianjmurrell) [inline lwgrp.spec]:
```suggestion
* Tue Jan 19 2021 Dalton A. Bohning <daltonx.bohning@intel.com> - 1.0.3-1
```

**Review by brianjmurrell — CHANGES_REQUESTED** [review]

**Review by brianjmurrell — APPROVED** [review]

**Review by brianjmurrell — APPROVED** [review]

**Comment by daltonbohning** [comment]
Order of dependencies:
lwgrp dot version update - This PR
dtcmp dot  version update - (depends on lwgrp) https://github.com/daos-stack/dtcmp/pull/3
mpifileutils minor version update - (depends on dtcmp) https://github.com/daos-stack/mpifileutils-pkg/pull/9

**Comment by daltonbohning** [comment]
@brianjmurrell I'm unable to add `daos-stack/release-engineering` to this PR, but it is ready to merge, followed by the dependent PRs.

**Comment by brianjmurrell** [comment]
> @brianjmurrell I'm unable to add `daos-stack/release-engineering` to this PR

Fixed.  It's a shame that GitHub doesn't allow a template projects can model when being created.  Instead we have to remember (or compare with another project) and set forty-eleven configuration items in every new project created.

**Pull request merged**